modules/rosa-cluster-autoscaler-cli-interactive.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-autoscaling.adoc
+// * osd_cluster_admin/osd-cluster-autoscaling.adoc
+
+:_content-type: REFERENCE
+[id="rosa-cluster-autoscale-cli-interactive_{context}"]
+= Enable autoscaling using the interactive mode with the ROSA CLI
+
+When running inside an interactive shell, the interactive autoscaling mode displays. This method has the benefit of providing more information about the different configurable parameters. It also does basic checks and preflight validations, meaning if a provided value is invalid, then it asks again for a valid input.
+
+.Procedure
+
+. To force interactive mode when not using an interactive shell, run the following command:
```

Comment:
The other "Enabling autoscaling modules do not have an intro paragraph before the procedure.
```suggestion
```